src/content/desci/index.md

```diff
@@ -8,77 +8,268 @@ sidebar: true
+Publishing networks allow research to be published, reviewed, elevated, and replicated without input from a controlling authority or company. Imagine never having to pay to access a research object. Imagine researchers getting paid directly for their work instead of through an intermediary. Imagine a marketplace of people and entities that provide review, curation, and replication services, instead of an oligopoly of a few major conglomerates.
 
-### ...what else?
+**Contributors to Web3**
 
-Derp derp derp
+The people and institutions who have been brought on board through DeSci's tooling have a lot of experience and knowledge in different areas. They've been working on similar problems to those facing Web3 for a long time. DeSci is making tools that will make their lives easier and let them keep working on the hardest and most rewarding problems facing society.  Having economists and mathematicians, to social psychologists and historians, with centuries of knowledge from hundreds of thousands of academics can help build the foundation of the macro-systems we're trying to create. In doing this, we're inviting them into the trustless world to keep doing what they're good at: solving the most challenging and fulfilling problems society has to offer.
 
-## How does DeSci work?
 
-Derp derp derp
-How does it work? How is Ethereum/blockchain technology used?
+## Innovation
+
+The systems of decentralized science require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These new methods are applied to science but can be used for other purposes as well. From rich on-chain research objects to IP-NFTs and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward. The Cambrian explosion of projects and concepts since inception creates an environment of rapid experimentation where anything is possible.
+
+
+## Legitimacy
+
+The crypto world is threatened by the possibility of extreme regulation. Web3 needs examples of blockchain use-cases that are easily understood and palatable, demonstrating a capability to increase efficiency, efficacy, reliability, and public trust of critical systems. DeSci is perhaps the best positioned to develop these examples..
+
+
+## Scalable knowledge creation
+
+Current research on Web3 is conducted in a way that is not efficient or effective in producing reliable information. The scientific method is a much better way to distill knowledge, and academia is trained in using this method. DeSci is creating the tool stack needed to apply the scientific method to research on Web3.
```

Comment:
Something about this section doesn't sit quite right with me, and I think it is the `legitimacy` subsection feels too self serving. 

The DeSci value proposition must be that it makes science somehow more frictionless for scientists & public and the web3 stack uniquely has the right primitives to do that. If part of the value-prop of building science infra on Web3 is propping up Web3 that signals misaligned incentives to me.

Would like to hear more opinions on this.

I do, however, love the point about DeSci as a tool for applying the scientific method to its own base infrastructure.

Reply:
good point! would you be open to rewrite or change it?

Reply:
I agree that "DeSci provides legitimacy" can be a dangerous proposition. I do not think crypto doesn't have legitimacy already, however I think legitimacy is in question in the eyes of many. Drugs, ponzi, bubbles, NFTs, memes, etc. 

Science is a system that almost everyone can get behind. Demonstrating the use case of DLT as it applies to science might convince many doubters that yes, there are interesting growing pains to this technology, but it can also make science better... more frictionless etc.

I would propose this change to the legitimacy section:

The crypto world is threatened by the possibility of extreme regulation. Web3 needs examples of blockchain use-cases that are easily understood and palatable, and that demonstrate a capability to increase efficiency, efficacy, reliability, and public trust of critical societal systems. DeSci is perhaps the best positioned to develop these examples.

Reply:
sure - I will let it sit for a little while to see if anyone else has opinions to share first.

Reply:
I think the other issue I have with this is that if Web3 can enjoy a legitimacy boost from the success of DeSci, the opposite is also true. DeSci is a massive risk area, because it is a nascent space that can still easily be co-opted by profiteers and shills. These are existential risks for DeSci, and there is contagion risk for wider Web3.